gapic/src/main/com/google/gapid/views/FramebufferView.java

```diff
@@ -511,8 +511,11 @@ private void updateUi() {
 
         image.setImage(attachment.getImage(widgets, this, this));
         label.setText(attachment.label);
-        image.requestLayout();
-        label.requestLayout();
+        // Skip 'loading' image layout changes until attachment selection image is available
+        if (attachment.image.hasFinished()) {
+          label.requestLayout();
+          image.requestLayout();
+        }
```

Comment:
I like this part. I wonder if not requesting the layout works reliably on every platform. Can you let me know where you've tested this and I can check the rest?

Code-wise, two changes, please:
1) call `label.requestLayout()` always, since if the text changes we don't want it to get cut off.
2) add a `isImageReady()` to `Attachment`, which will `return image != null && image.hasFinished()` and use it here instead of reaching into the internals of `Attachment`. This will avoid any potential `null` issues, which is why `image` is private in `Attachment` in the first place.